examples/asr/conf/carnelinet/carnelinet_384.yaml

```diff
@@ -0,0 +1,259 @@
+        se_repeat: ${model.model_defaults.se_repeat}
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: [13]
+        stride: [2]  # stride
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+        stride_last: true
+        residual_mode: "stride_add"
+
+      - filters: 384
+        repeat: ${model.model_defaults.repeat}
+        kernel: ["${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}","${model.model_defaults.kernel_size}"]
+        stride: [1]
+        dilation: [1]
+        dropout: ${model.model_defaults.dropout}
+        residual: true
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+
+      - filters: &enc_final 640
+        repeat: 1
+        kernel: ["${model.model_defaults.kernel_size}"]
+        stride: [1]
+        dilation: [1]
+        dropout: 0.0
+        residual: false
+        separable: ${model.model_defaults.separable}
+        se: ${model.model_defaults.se}
+        se_context_size: ${model.model_defaults.se_context_size}
+        se_repeat: ${model.model_defaults.se_repeat}
+
+
+  decoder:
+    _target_: nemo.collections.asr.modules.ConvASRDecoder
+    feat_in: *enc_final
+    num_classes: -1  # filled with vocabulary size from tokenizer at runtime
+    vocabulary: []  # filled with vocabulary from tokenizer at runtime
+
+  optim:
+    name: novograd
+    lr: 0.1
+
+    # optimizer arguments
+    betas: [0.8, 0.25]
+    weight_decay: 0.001
+
+    # scheduler setup
+    sched:
+      name: CosineAnnealing
+
+      # scheduler config override
+      warmup_steps: 1000
+      warmup_ratio: null
+      min_lr: 1e-5
+      last_epoch: -1
+
+trainer:
+  gpus: 0 # number of gpus
+  max_epochs: 100
+  max_steps: null # computed at runtime if not set
+  num_nodes: 1
+  accelerator: ddp
+  accumulate_grad_batches: 1
+  checkpoint_callback: false  # Provided by exp_manager
+  logger: false  # Provided by exp_manager
+  log_every_n_steps: 50  # Interval of logging.
+  val_check_interval: 1.0 # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
+  precision: 16 # Train with AMP to speed and memory efficiency
+  sync_batchnorm: false
+  benchmark: false
+
+exp_manager:
+  exp_dir: null
+  name: *name
+  create_tensorboard_logger: true
+  create_checkpoint_callback: true
+  checkpoint_callback_params:
+    monitor: "val_wer"
+    mode: "min"
```

Comment:
Can you add always_save_nemo = True flag here. We should make it the default.

Reply:
Done.